src/Symfony/Component/ErrorHandler/Resources/bin/patch-type-declarations

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env php
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+require __DIR__.DIRECTORY_SEPARATOR.'patch-type-declarations.php';
```

Comment:
let's use `/` instead of `DIRECTORY_SEPARATOR`

but I'm not sure we need to split the .php file from this one. Can we drop the .php one and merge it's content here? (and set this file in 755 mode)

Reply:
Yes, that works for me. I just replicated the `simple-phpunit` set-up :) (including directory separator)